package.json

```diff
@@ -35,6 +35,7 @@
     "copy-webpack-plugin": "^4.5.1",
     "css-loader": "^0.28.11",
     "grafana-sdk-mocks": "github:grafana/grafana-sdk-mocks",
+    "replace-in-file-webpack-plugin": "^1.0.6",
```

Comment:
This needs to be filed in alphabetical order.

Reply:
done